tests/lib/prepare-utils.sh

```diff
@@ -114,9 +116,17 @@ start_snapd_core_vm() {
     nested_wait_for_snap_command
 }
 
+get_arch() {
+    if os.query is-pc-amd64; then
+        printf amd64
+    else
```

Comment:
`elif os.query is-arm64` and an error in the `else`, just for it to break if we add another architecture.

Reply:
Done